web3.js/test/connection.test.ts

```diff
@@ -2077,6 +2077,36 @@ describe('Connection', () => {
     );
   });
 
+  it('get blocks between two slots', async () => {
+    await mockRpcResponse({
+      method: 'getBlocks',
+      params: [0, 10],
+      value: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    });
+
+    const blocks = await connection.getBlocks(0, 10);
+    expect(blocks).to.have.length(10);
```

Comment:
This could still fail... how about you first request `connection.getSlot` to see what slots are expected and ensure that we have at least that many?

Reply:
Actually I tested it locally and it used to have 42 blocks but it is 24 now. However, I'll implement what you suggested.

Reply:
Hope this change is OK :)

Reply:
Looks great, could you apply the same fix to the other test please?

Reply:
Btw, I have this:

`expect(blocks).to.have.lengthOf.greaterThanOrEqual(latestSlot);`

However there may be less than last slot number slots, right? So, I need to change that line.

Reply:
Also, another test at connection.test.ts has these lines:

```
    while ((await connection.getSlot()) <= 0) {
      continue;
    }
```

should I follow the same approach?

Reply:
> However there may be less than last slot number slots, right? So, I need to change that line.

In this case, with one validator, there should always be a block for every slot

> should I follow the same approach?

Sure, sounds great!

Reply:
I used to think that block numbers may not be sequential.